catalog/app/containers/Bucket/WithPackagesSupport.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+
+import Message from 'components/Message'
+import * as BucketConfig from 'utils/BucketConfig'
+import * as Config from 'utils/Config'
+import StyledLink from 'utils/StyledLink'
+
+const ADD_BUCKET_DOCS = 'https://docs.quiltdata.com/catalog/admin#buckets'
```

Comment:
Probably, code can be more changeable by using constants/urls:

```ts
import { docs } from 'constants/urls'

const ADD_BUCKET_DOCS = `${docs}/catalog/admin#buckets`
```

Reply:
well yeah probably